.github/workflows/build.yml

```diff
@@ -45,7 +45,9 @@ jobs:
         env:
           cache-name: cache-node-modules
         with:
-          path: ~/.npm
+          path: |
+            ~/.npm
+            node_modules
```

Comment:
While I'm debugging the test failure (have to do it on GH, it doesn't fail on my local Windows), I noticed that the `node_modules` directory isn't cached. Is this on purpose? Or an oversight?

Reply:
Possibly an oversight? Let's keep it in and see if it breaks things

Reply:
I also added an attempt to cache the Windows equivalent of `~/.npm`, though I'm not 100% sure it worked. But the lack of caching here is big part of the reason that Windows build is so much slower.